ghost/member-attribution/lib/referrer-translator.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * Translates referrer data into source and medium
+ */
+class ReferrerTranslator {
+    /**
+     *
+     * @param {Object} deps
+     * @param {Object} deps.urlUtils
+     */
+    constructor({urlUtils}) {
+        this.urlUtils = urlUtils;
```

Comment:
Yea that works as well, the reason I had left `urlUtils` was in case we'd want to extend the logic in future, will switch to simpler `adminUrl` for now and we can switch back again later. 👍

Reply:
Updated deps to pass in `adminUrl` and `siteUrl` directly ✔️